flutter/shell/platform/tizen/flutter_tizen_view.h

```diff
@@ -25,10 +25,14 @@ namespace flutter {
 
 class FlutterTizenView : public TizenViewEventHandlerDelegate {
  public:
-  FlutterTizenView(std::unique_ptr<TizenViewBase> tizen_view);
+  FlutterTizenView(FlutterViewId view_id,
+                   std::unique_ptr<TizenViewBase> tizen_view);
 
   virtual ~FlutterTizenView();
 
+  // Get the view's unique identifier.
+  FlutterViewId view_id() const;
```

Comment:
nit: For simple accessors,
```suggestion
  FlutterViewId view_id() const {
    return view_id_;
  }
```

Reply:
done!